Source/WebCore/platform/graphics/texmap/TextureMapperLayer3DRenderingContext.cpp

```diff
@@ -30,39 +30,150 @@
 
 #include "ClipPath.h"
 #include "FloatPlane3D.h"
+#include "FloatPolygon3D.h"
+#include "GeometryUtilities.h"
 #include "TextureMapperGPUBuffer.h"
 #include "TextureMapperLayer.h"
-#include <wtf/Deque.h>
+#include <numeric>
+#include <wtf/StdSet.h>
```

Comment:
Remove this line. StdSet is no longer used.

Reply:
Fixed